packages/astro/test/fixtures/plain-markdown/astro.config.mjs

```diff
@@ -5,4 +5,7 @@ export default {
   buildOptions: {
     sitemap: false,
   },
+  markdownOptions: {
+    defaultLayout: '../layouts/default.astro'
```

Comment:
needs to relative to src directory (or root directory). `../` doesn't make sense in this context.